### DISCUSSION TITLE
Allow Users to Select Installation Location on WIndows

### DISCUSSION THREAD

**Discussion by vehm** [scalar/scalar#3443] [Ideas]
Many modern applications are addressing issues relating to write permissions on Windows by installing software by default in the `%APPDATA%` directory, sometimes without providing users the option to change the installation directory.

Please consider allowing users to select an installation directory rather than defaulting to a fixed location like `%APPDATA%` (which is user-scoped), or at least providing a separate installation option for system-wide installations.

**Comment by hanspagel** [comment]
For our upcoming Windows app? Cool!

**Reply by vehm** [reply]
Yeah! Not trying to name names, but it irks me that another popular API client installs itself inside of `%APPDATA%` without any indication of it doing so nor any option to install it elsewhere. It would be nice to have the choice with Scalar!
